docs/build-parachains.md

```diff
@@ -0,0 +1,174 @@
+---
+id: build-parachains
+title: Parachain Development
+sidebar_label: Parachain Development
+---
+
+This section will cover the motivation to build a parachian or parathread, the tools available
+to facilitate this, and the steps to test, and finally launch your network on Polkadot!
+
+## Why Create a Parachain?
+
+Parachains can [lease the security](learn-security) of the Polkadot network by bonding
+[DOT](learn-DOT) for a parachain slot. This means that the social costs of building a community
+around your project and convincing validators to participate in your network security are reduced.
+Polkadot has strong security, and decentralized application projects wishing to benefit from this
+security would want to become a parachain to share in that pooled security. For more information
+on the mechanic of leasing a parachain slot through a candle auction see [here](learn-auction).
```

Comment:
Maybe better to say something along the lines of "More information
on the mechanic of leasing a parachain slot is available on the [Parachain Slots Auction](learn-auction) page.